{
	"id": "CVE-2018-12563",
	"description": "An issue was discovered in Linaro LAVA before 2018.5.post1. Because of support for file: URLs, a user can force lava-server-gunicorn to download any file from the filesystem if it's readable by lavaserver and valid yaml.",
	"scope": "local",
	"packages": [
		{
			"name": "lava",
			"status": "resolved",
			"urgency": "not yet assigned",
			"fixed_version": "2018.5.post1-1",
			"repository": [
				{
					"name": "sid",
					"version": "2024.09-1"
				}
			]
		}
	]
}
